server/common.ts

```diff
@@ -43,11 +34,13 @@ class Config implements IConfig.Main {
 		workflowReleaseSummary: null,
 		cookieMaxAge: 1000 * 60 * 60 * 24 * 30 * 6, // 6 months
 		cookieSecureOnly: false,
-		mongoURL: "mongodb://localhost/",
-		passwordResetExpiration: 1000 * 60 * 60, // 1 hour
+		mongoURL: "mongodb://localhost/registration",
 		defaultTimezone: "America/New_York"
 	};
-	public admins: string[] = [];
+	public admins = {
+		domains: ["hack.gt"],
```

Comment:
I think this should be defined in an env variable / config.  If anyone outside HackGT were to deploy registration, anyone with an @hack.gt email (aka us) would have admin access to their instance if they weren't careful, since this isn't documented in the README:
`ADMIN_EMAILS | A JSON array of the emails of the users that you want promoted to admin status when they create their account (default: none)`

Given the above, there shouldn't be any default admin users/emails unless defined in the environment variables or config.json.

Reply:
Really good catch! Left that in there from when I was testing. I'm replacing it with an empty array because overriding it with a config setting is already supported